README.md

```diff
@@ -51,3 +51,7 @@ names(lintr::linters_with_tags(tags = NULL))
 You can also run lintr during continuous integration or within your IDE or text editor. See `vignette("continuous-integration")` and `vignette("editors")` for more details.
 
 Without further configuration, this will run the [default linters](https://lintr.r-lib.org/reference/default_linters.html). See `vignette("lintr")` to learn how to modify these defaults.
+
+## Code of Conduct
+
+Please note that the lintr project is released with a [Contributor Code of Conduct](https://lintr.r-lib.org/CODE_OF_CONDUCT.html). By contributing to this project, you agree to abide by its terms.
```

Comment:
Note that GHA to detect link rot is going to fail because `https://lintr.r-lib.org/CODE_OF_CONDUCT.html` will be rendered only **after** this PR is merged into the `main` and the pkgdown website is built.